### PR TITLE
feat(js): Add npm script info

### PR DESCRIPTION
**Pull request by Teapane**
#18

### CODE REVIEW

**Comment by c0** [comment]
:+1: 

``` json
  "scripts-info": {
    "build": "Build JavaScript assets once",
    "start": "Install and boot a local server at localhost:3000",
    "lint": "Run eslint and report errors",
    "lint:fix": "Run eslint and attempt to fix errors",
    "info": "See `npm run info`",
    "test": "Run mocha tests",
    "test:watch": "Same as `test`, but runs tests when files change",
    "test:cov": "Run tests and report coverage",
    "styleguide": "Build the styleguide from the React components",
    "styleguide:watch": "Same as `styleguide`, but continues watching for changes"
  },
```

**Comment by c0** [comment]
:boom:

File: package.json [hunk 1]
```diff
@@ -34,6 +35,7 @@
     "history": "1.12.6",
     "immutable": "^3.7.5",
     "isomorphic-fetch": "^2.2.0",
+    "npm-scripts-info": "^0.1.0",
```
Comment (c0) [inline package.json:38]: Oh, I just realized this is in `dependencies`. What do you think about moving it to `devDependencies`?

**Comment by Teapane** [comment]
@c0 agreed. I can move it; however, see conversation in #18...

**Pull request merged**